spec.bs

```diff
@@ -372,10 +370,8 @@ To <dfn>append private state token issue request headers</dfn> given a [=/reques
     Issue: This probably needs to be its own algorithm, how is "most recent" defined here?
 
  1. Let |maxSize| be |issuer|'s [=max batch size=] or the global limit of 100, whichever is smaller.
- 1. Let |numTokens| be the right number of tokens considering issuer batch size and current
-     number of tokens from the issuer.
-
-    Issue: How does one determine the right number of tokens?
+ 1. Let |numTokens| be the number of tokens to request using the minimum of an [=implementation-defined=]
+     limit on the number of tokens and the issuer's batch size.
```

Comment:
Is this the `|issuer|'s [=max batch size=]`?

Reply:
Yeah, the one specified from above.